src/shared/components/ncEditor/NcEditor.vue

```diff
@@ -72,8 +72,15 @@ export default {
 	watch: {
 		text(value) {
 			if (value.trim() !== this.localValue.trim()) {
-				this.editor.setContent(value, false)
 				this.localValue = value
+
+				// reset editor if content is empty
+				// otherwise the (empty) content will not be updated
+				if (value === '') {
+					this.setupEditor()
+				} else {
+					this.editor.setContent(value, true)
```

Comment:
Wondering about the second parameter now, there should only be one https://github.com/nextcloud/text/blob/a0f9aa64c10200479c6c1643a0ce33de221b5a76/src/editor.js#L76